webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTableHelper.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
+import CheckIcon from '@patternfly/react-icons/dist/js/icons/check-icon';
+import { Link } from 'react-router-dom';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const formatSourceLink = (hostAttrs, variable) => {
+  const value = `${variable.currentValue.element}: ${variable.currentValue.elementName}`;
+  // eslint-disable-next-line prefer-const
+  let { element, meta } = variable.currentValue;
+
+  const knownMatchResources = ['os', 'hostgroup', 'domain'];
+
+  if (!knownMatchResources.includes(element)) {
+    return value;
+  }
+
+  if (element === 'os') {
+    element = 'operatingsystem';
```

Comment:
does it make sense to store the data in the redux's store?

Reply:
Doing that would not bring any benefits.

Reply:
I mean, the data comes from the server, like we have the API middleware and selectors for fetching, storing, cleaning, formatting, and manipulate data. I wrote a suggestion in another comment.